src/news/news.js

```diff
@@ -0,0 +1,51 @@
+import './news.scss';
+
+export default class News {
+  constructor () {
+    this.showNews = this.showNews.bind(this);
+  }
+
+  async getNewsAsync(url) {
+    let response = await fetch(url);
+    let data = await response.json();
+    return data;
+  }
+
+  showNews (event) {
+    const urlNews = `https://newsapi.org/v2/everything?sources=${event.target.getAttribute('id')}&pageSize=10&apiKey=83c8aebd6ec444179f992b3fc49b9b3f`;
```

Comment:
Ufff, quite a heavy line. Maybe worth to split it a bit

Reply:
Also, such params as pageSize can come from the configuration (let say json file)

Reply:
done